packages/hdwallet-metamask/src/ethereum.ts

```diff
@@ -63,7 +63,7 @@ export async function ethSendTx(msg: core.ETHSignTx, ethereum: any, from: string
       data: msg.data,
       chainId: msg.chainId,
       nonce: msg.nonce,
-      gasLimit: msg.gasLimit,
+      gas: msg.gasLimit,
```

Comment:
Can you add a comment here with a link to a source that documents this behavior for future reference?

Reply:
Done